dart/tool/dart/generate_dart.dart

```diff
@@ -454,13 +457,20 @@ const String undocumented = 'undocumented';
 /// This is useful for handling the results of the Stdout or Stderr events.
 String decodeBase64(String str) => utf8.decode(base64.decode(str));
 
-Object createServiceObject(dynamic json) {
+// Returns true if a response is the Dart `null` instance.
+bool _isNullInstance(Map json) => ((json['type'] == '@Instance') &&
```

Comment:
needs a `dartfmt`?

Reply:
I ran `dartfmt` on this and that's what it gave me * shrugs *